.circleci/config.yml

```diff
@@ -90,7 +90,11 @@ jobs:
           # allowed. Adding `-A dead_code` until state_machine_future is removed.
           command: |
             cargo fmt -- --check
-            cargo clippy --all --all-targets --all-features -- -D warnings --deny=clippy::dbg_macro -A dead_code
+            # clippy for rust 1.64+ tags the proc_macro generated "start_in" from
+            # state_machine_future as "dead_code". There's no way to isolate just
+            # those calls, so we have to disable dead_code checks globally.
+            # `--allow=dead_code` must be removed when we remove `state_machine_future`
+            cargo clippy --all --all-targets --all-features -- -D warnings --deny=clippy::dbg_macro --allow=dead_code
```

Comment:
Huh. I remember trying that and it didn't work. Clippy still flagged it, presumably because the macro expanded it before the tag was read? I'll retry because that might have been a cargo bug.